docs/tutorials/chain-deploying.md

```diff
@@ -1,16 +1,16 @@
 ---
 
 layout: single
-title: "Multi Node Chains"
+title: "Tutorials | Multi Node Chains"
 aliases:
-  - /docs
+  - /docs/chain-deploying
 menu:
   tutorials:
     weight: 5
 
 ---
 
-## Introduction
+# Introduction
```

Comment:
It absolutely does - the problem is how the ToCs get automatically generated. Before fixing the H1s across the site we'll need to reprogram how the Table of Contents get formed - chapter links only appear if there are several H1s in the content, but I'd rather have a page which is good for users to read than good for search engines to read - at least for next week or two while we sort that out.